modules/core/04-channel/keeper/grpc_query.go

```diff
@@ -541,6 +541,41 @@ func (q Keeper) NextSequenceReceive(c context.Context, req *types.QueryNextSeque
 	return types.NewQueryNextSequenceReceiveResponse(sequence, nil, selfHeight), nil
 }
 
+// NextSequenceSend implements the Query/NextSequenceSend gRPC method
+func (k Keeper) NextSequenceSend(c context.Context, req *types.QueryNextSequenceSendRequest) (*types.QueryNextSequenceSendResponse, error) {
+	ctx := sdk.UnwrapSDKContext(c)
+	if req == nil {
+		return nil, status.Error(codes.InvalidArgument, "empty request")
+	}
+
+	if err := validategRPCRequest(req.PortId, req.ChannelId); err != nil {
+		return nil, err
+	}
```

Comment:
As per #3422, we should move the unwrapping after the error handling, this is similar to how other queries are doing it (i.e `ChannelClientState` and `ChannelConsensusState`):

```suggestion
	if req == nil {
		return nil, status.Error(codes.InvalidArgument, "empty request")
	}

	if err := validategRPCRequest(req.PortId, req.ChannelId); err != nil {
		return nil, err
	}
	
	ctx := sdk.UnwrapSDKContext(c)
```

Reply:
Yes. Thank you